packages/web3-eth-contract/src/contract.ts

```diff
@@ -567,7 +567,6 @@ export class Contract<Abi extends ContractAbi>
 		 * The byte code of the contract.
 		 */
 		data?: HexString;
-		input?: HexString;
```

Comment:
The documentation has to be updated for this :smile:. 
I found a use in the documentation here: https://github.com/web3/web3.js/blob/b88f2e8ac40b5718a8dbf8ac6701155442ce1bba/docs/docs/guides/sign_and_send_tx/local_wallet.md#L65-L66 and there possibly more.